CHANGELOG.md

```diff
@@ -4,6 +4,35 @@ All notable changes to this project will be documented in this file.
 
+## [Preview Release 2.0.0-preview1.20084.1] - 2020-03-24
+
+### Added
+- Added support for capturing EventSource traces in .NET Framework, .NET Core, and .NET Standard applications [#399](https://github.com/dotnet/SqlClient/pull/399) [#461](https://github.com/dotnet/SqlClient/pull/461) [#479](https://github.com/dotnet/SqlClient/pull/479) [#483](https://github.com/dotnet/SqlClient/pull/483) [#484](https://github.com/dotnet/SqlClient/pull/484)
+- Added support for Cross-platform TCP Keep Alive applicable to .NET Core 3.1+ applications [#395](https://github.com/dotnet/SqlClient/pull/395)
+- Added support for enabling Managed networking implementation on Windows applicable to .NET Core and .NET Standard applications [#477](https://github.com/dotnet/SqlClient/pull/477)
+- Added `RowsCopied` property in `SqlBulkCopy` to expose count of copied rows [#409](https://github.com/dotnet/SqlClient/pull/409)
+- Added "NeutralResourcesLanguage" attribute for .NET Framework assembly [#433](https://github.com/dotnet/SqlClient/pull/433)
+- Added caching for invariant culture check result [#376](https://github.com/dotnet/SqlClient/pull/376)
+- Added cached `SqlReferenceCollection.FindLiveReaderContext` objects [#380](https://github.com/dotnet/SqlClient/pull/380)
+
+### Fixed
+- Fixed Access Token behavior in connection pool to perform string comparison [#443](https://github.com/dotnet/SqlClient/pull/443)
+- Fixed concurrent connection speed issues when connecting with Azure Federated Authentication modes in .NET Core [#466](https://github.com/dotnet/SqlClient/pull/466)
+- Fixed issues with `Password` persistence in Connection String [#453](https://github.com/dotnet/SqlClient/pull/453)
+
+### Changes
+- Updated all driver assemblies to become CLS Compliant [#396](https://github.com/dotnet/SqlClient/pull/396)
+- Updated Bulk Copy error messages to also include Column, Row and non-encrypted Data information [#427](https://github.com/dotnet/SqlClient/pull/437)
+- Updated error messages for "Always Encrypted - Secure Enclaves" to handle 'Attestation Protocol' and fixed typos [#421](https://github.com/dotnet/SqlClient/pull/421) [#397](https://github.com/dotnet/SqlClient/pull/397)
+- Removed sync over async in `SNINpHandle.EnableSsl` [#474](https://github.com/dotnet/SqlClient/pull/474)
+- Changed non-generic `ArrayList` to `List<T>` in `SqlBulkCopy` [#457](https://github.com/dotnet/SqlClient/pull/457)
+- Multiple performance improvements [#377](https://github.com/dotnet/SqlClient/pull/377) [#378](https://github.com/dotnet/SqlClient/pull/378) [#379](https://github.com/dotnet/SqlClient/pull/379) 
+
+### Breaking Changes
+- The driver will now perform Server Certificate validation when TLS encryption enforced by target Server, effective automatically on Azure connections [#391](https://github.com/dotnet/SqlClient/pull/391)
+- `SqlDataReader.GetSchemaTable()` now returns empty `DataTable` instead of returning `null` [#419](https://github.com/dotnet/SqlClient/pull/419)
```

Comment:
```suggestion
- `SqlDataReader.GetSchemaTable()` now returns an empty `DataTable` instead of returning `null` [#419](https://github.com/dotnet/SqlClient/pull/419)
```